packages/pf-codemods/README.md

```diff
@@ -71,7 +71,7 @@ In:
 Out:
 
 ```jsx
-<Card  />
+<Card />
```

Comment:
prettier ran?

Reply:
Must have, I'll go back and add spaces

Reply:
I'm going to leave in the single quotes changing to double quotes because that's what the JSX should look like